.github/ISSUE_TEMPLATE/build_error.yml

```diff
@@ -0,0 +1,60 @@
+name: "\U0001F4A5 Build error"
+description: Some package in Spack didn't build correctly
+title: "Installation issue: "
+labels: [build-error]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to report this build failure. To proceed with the report please:
+        1. Title the issue "Installation issue: <name-of-the-package>".
```

Comment:
It seems like `<name-of-the-package>` is being treated as an unknown HTML tag and not displaying. Let's remove the angle brackets and use italics instead?

Reply:
Good catch. Fixed using `an inline code block`